insights/tests/client/apps/test_compliance.py

```diff
@@ -123,3 +124,32 @@ def test_run_scan_fail(config, call):
     with raises(SystemExit):
         compliance_client.run_scan('ref_id', '/nonexistent', output_path)
     call.assert_called_with("oscap xccdf eval --profile ref_id --results " + output_path + ' /nonexistent', keep_rc=True, env=env)
+
+
```

Comment:
What about a test for malformed API response. i.e. a response missing 'attributes' or 'tailored'. This client might be run against the wrong version of our backend, so I think it'd be good to handle those cases.

Reply:
Updated, see ` test_tailored_file_is_not_downloaded_if_tailored_is_missing`